account_invoice_validate_send_email/models/account_invoice.py

```diff
@@ -88,7 +88,7 @@ def action_invoice_open(self):
     def action_send(self):
         # send notification email for follower
         self.ensure_one()
-        if self.type == "out_invoice" or "out_refund":
+        if self.type == "out_invoice" or self.type == "out_refund":
```

Comment:
```suggestion
        if self.type in ("out_invoice", "out_refund"):
```

Reply:
Thank you, now succeed automatic mail sent.